.github/ISSUE_TEMPLATE/enhancement---suggestion.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Enhancement / Suggestion
+about: Suggest an idea for this project
+title: ''
+labels: status/need-triage, type/enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+<!--- Make sure to follow the [Contribution Guidelines](https://github.com/reactor/.github/blob/master/CONTRIBUTING.md), -->
+<!--- notably for [security-related issues](https://pivotal.io/security) -->
+<!-- Questions should be asked on [Gitter](https://gitter.im/reactor/reactor) or [StackOverflow](https://stackoverflow.com/questions/tagged/project-reactor). -->
```

Comment:
We have different Gitter rooms per project and also different StackOverflow tags

Reply:
done